frontend/src/pages/IstioConfigNew/IstioConfigNewPage.tsx

```diff
@@ -15,6 +15,7 @@ import {
   HelperTextItem,
   TextInput
 } from '@patternfly/react-core';
+import { PencilAltIcon } from '@patternfly/react-icons/dist/esm/icons/pencil-alt-icon';
```

Comment:
Since we have KialiIcon wrapper component, I think we should use our own component instead of patternfly one directly.